### PR TITLE
Update license conditions to apache-2

### PR DESCRIPTION
**Pull request by PaulChana**
Updates the license to apache-2 and adds the contributor covenant and a couple of small things from legal (may need to change again once ive confirmed exact wording with legal dept)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+For answers to common questions about this code of conduct, see the FAQ at
+[https://www.contributor-covenant.org/faq][FAQ]. Translations are available
+at [https://www.contributor-covenant.org/translations][translations].
+
+[homepage]: https://www.contributor-covenant.org
+[v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
+[Mozilla CoC]: https://github.com/mozilla/diversity
+[FAQ]: https://www.contributor-covenant.org/faq
+[translations]: https://www.contributor-covenant.org/translations
+
```
Comment (musomatt) [inline CODE_OF_CONDUCT.md:133]: all these 🇺🇸 spellings tho

**Review by musomatt — APPROVED** [review]
Looks good, are we adding this to all repositories and just using this one as a test?

**Comment by PaulChana** [comment]
> Looks good, are we adding this to all repositories and just using this one as a test?

@musomatt Only public repositories. But I have no objection to having Contributor covenant in every repo. It basically says "Be a nice person", which I cant imagine many people would object to

**Comment by musomatt** [comment]
> @musomatt Only public repositories. But I have no objection to having Contributor covenant in every repo. It basically says "Be a nice person", which I cant imagine many people would object to

ah yeah, forgot this repo is public.

**Review by willm — APPROVED** [review]
Nice one @PaulChana

**Pull request merged**